app/core/forms.py

```diff
@@ -68,8 +68,8 @@ def __init__(self, *args, **kwargs):
 
 
 class RegistrationForm(forms.Form):
-    first_name = forms.CharField(label=_('First_name'), max_length=30)
-    last_name = forms.CharField(label=_('Last_name'), max_length=30)
+    first_name = forms.CharField(label=('First_name'), max_length=30)
+    last_name = forms.CharField(label=('Last_name'), max_length=30)
```

Comment:
This can't be right.. Why are you removing the translation here? Are you even using the label in the template?

Reply:
I thought that I only have to translate the labels in the template. Is that wrong?